server.js

```diff
@@ -20,7 +29,8 @@ app.post('/bar', (req, res) => {
   serverObj.push(example1)
 })
 
-app.get('/foo', (req, res) => {
-  const test = {foo : 'bar'}
-  res.send(test)
-})
\ No newline at end of file
+app.get('/foo', async (req, res) => {
+  const template = 'SELECT * from weddinglist'
+  const response = await pool.query(template)
+  res.json(response.rows)
```

Comment:
[The client](https://github.com/jiwonkim19/weddinginvite/blob/b03c9eb703d0609c22f073d1cce2cb29c92cd53d/client/src/components/Foo.jsx#L11) is expecting the response to have a `.bar` attribute. Can you update the response to fit that requirement?

Reply:
Oops. Update it. thanks